_assets/js/laws-regs-parser.js

```diff
@@ -83,15 +87,15 @@ function buildBtns(i, divType) {
         btn.setAttribute('aria-label', btnType);
         btn.href = '#';
         if (btnType === 'Share') {
-            btn.innerHTML = '<span class="copied-link text-no-underline" style="display:none;">Copied link</span><svg ' + gaEventName + ' class="usa-icon share-icon usa-tooltip" data-position="bottom" focusable="false" role="img"><title>copy link</title><use xlink:href="/assets/img/sprite.svg#link"></use></svg>';
+            btn.innerHTML = '<span class="copied-link text-no-underline" style="display:none;">Copied link</span><svg title="Copy link" ' + gaEventName + ' class="usa-icon share-icon usa-tooltip" data-position="bottom" focusable="false" role="img"><title>Copy link</title><use xlink:href="/assets/img/sprite.svg#link"></use></svg>';
             btn.addEventListener('click', (e) => { shareLink(e, divType) });
         } else if (btnType === 'Copy') {
-            btn.innerHTML = '<span class="copied text-no-underline" style="display:none;">Copied text</span><svg ' + gaEventName + ' class="usa-icon copy-icon usa-tooltip" data-position="bottom" focusable="false" role="img"><title>copy text</title><use xlink:href="/assets/img/sprite.svg#content_copy"></use></svg>';
+            btn.innerHTML = '<span class="copied text-no-underline" style="display:none;">Copied text</span><svg title="Copy text" ' + gaEventName + ' class="usa-icon copy-icon usa-tooltip" data-position="bottom" focusable="false" role="img"><title>Copy text</title><use xlink:href="/assets/img/sprite.svg#content_copy"></use></svg>';
             btn.addEventListener('click', (e) => { copyText(e, divType) });
             btn.addEventListener('mouseover', (e) => { highlightText(e, divType) });
             btn.addEventListener('mouseleave', (e) => { unHighlightText(e, divType) });
         } else if (btnType === 'Print') {
-            btn.innerHTML = '<svg title="print" ' + gaEventName + ' class="usa-icon usa-tooltip" data-position="bottom" focusable="false" role="img"><title>print</title><use xlink:href="/assets/img/sprite.svg#print"></use></svg>';
+            btn.innerHTML = '<svg title="Print" ' + gaEventName + ' class="usa-icon usa-tooltip" data-position="bottom" focusable="false" role="img"><title>Print</title><use xlink:href="/assets/img/sprite.svg#print"></use></svg>';
```

Comment:
maybe this would be better as a template string, so it could be multiline / help with interpolation (since html whitespace shouldn't matter here)?
```suggestion
            btn.innerHTML = `
              <svg
                title="Print"
                ${gaEventName}
                class="usa-icon usa-tooltip"
                data-position="bottom"
                focusable="false" role="img">
                  <title>Print</title>
                  <use xlink:href="/assets/img/sprite.svg#print">
                  </use>
                </svg>
              `;
```